app/controllers/bulk_export_controller.rb

```diff
@@ -1,4 +1,5 @@
 class BulkExportController < ApplicationController
+  before_action :authorized?, only: [:index, :show, :download]
```

Comment:
I don't think we want download in this list -- project owners and staff members (not just admins) need to be able to download bulk exports.